src/ivq.tsx

```diff
@@ -142,6 +154,18 @@ export class Ivq extends KalturaPlayer.core.BasePlugin {
     }
   };
 
+  private _serOverlay = (fn: Function) => {
```

Comment:
Not sure I understand what this does

Reply:
typo, fixed.
Method set IVQ Overlay that allow us remove the overlay component once we need it.